packages/api-explorer/src/Doc.jsx

```diff
@@ -299,15 +353,15 @@ class Doc extends React.Component {
               {this.renderSchemaTab()}
             </div>
             <div
+              className={'expandable'}
```

Comment:
This class name is not used

Reply:
is to simplify testing. It would be almost impossible to target that component